apps/meteor/app/mailer/server/api.ts

```diff
@@ -135,7 +135,13 @@
 });
 
 export const checkAddressFormat = (adresses: string | string[]): boolean =>
-	([] as string[]).concat(adresses).every((address) => validateEmail(address));
+	([] as string[]).concat(adresses).every((address) => {
+		const parts = address.split('<');
+		if (parts.length < 2) {
+			return validateEmail(address);
+		}
+		return validateEmail(parts[1].replace('>', ''));
```

Comment:
@Teak-Rosewood isn't better to include this pattern in the regex instead of breaking down the validation function?

Reply:
@hugocostadev im a bit lost. 

essentially the function that calls the validate email gets two main types of formats, one in the form of '${user.name||user.username}<${email.address}>' or the other in the form of '${email.address}', depending on weather the string contains a < I am splitting the strings and validating only the email portion of it, and if the < doesn't exist the function assumes that only the email has been given without the username. 

changing the regular expression to contain would be an easy fix but it allow users to have emails containing < which ideally should be an invalid email. 

the GitHub code checker is telling me that I'm not replacing all the '>'s in the string which I don't see an issue with as an email shouldn't contain an '>' either way and that email would fail the email validation test.

According to what you would prefer I can just change the regular expression to account for a < and > but I feel that would be counterintuitive as it might allow some invalid emails to be registered/

Reply:
also, i have only changed the regular expression in the validation function. this is another function which failed the test as the previous validation function was validating wrongly. the  function previously didn't account for the format I mentioned above while validating emails. 
so technically it would return a true for 
```
'name<name@gmail.com>'
```
when ideally only name@gmail.com should be validated

Reply:
the new commit fixes this issue by replacing all '>' with ''